content/en/users/getting-started/_index.md

```diff
@@ -3,9 +3,144 @@ title: "Getting Started"
 linkTitle: "Getting Started"
 type: docs
 weight: 5
-draft: true
 description: >
-  Introduction to using EGI services
+  Introduction to EGI FedCloud
 ---
 
-TBD
+## Overview
+
+EGI is a federation of computing and storage resource providers
+united by a mission to support research and innovation.
+
+The resources in the EGI Federated Cloud (FedCloud) are offered by
+Infrastructure-as-a-Service (IaaS) [service providers](https://www.egi.eu/federation/egi-federated-cloud/)
+that either run their own [data centers](https://www.egi.eu/federation/data-centres/)
+or rely on community, private and/or public cloud services. These service
+providers offer:
+
+- **Single Sign-On** via [EGI Check-in](https://www.egi.eu/services/check-in/)
+  allows users to login with their institutional (community) credentials
+- **Global image catalogue** at [AppDB](https://appdb.egi.eu) with
```

Comment:
If you remove this one, then I think it should cover all services of EGI (SSO, discovery, accounting, A/L monitoring is available for all of them)

Reply:
I do not understand